features/features.go

```diff
@@ -309,3 +252,57 @@ func FromContext(ctx context.Context) *Features {
 	}
 	return nil
 }
+
+func (f *Features) Adjust() ([]string, error) {
+	warningStrs := make([]string, 0)
+	
+	v := reflect.ValueOf(f).Elem()
+	t := v.Type()
+
+	for i := 0; i < t.NumField(); i++ {
+		field := t.Field(i)
+		value := v.Field(i)
+		version := field.Tag.Get("enabled_in_version")
+		if len(version) == 0 {
+			continue
+		}
+		majorVersion, minorVersion := ParseVersion(field.Tag.Get("enabled_in_version"))
+		if versionAtLeast(*f, majorVersion, minorVersion) && value.Kind() == reflect.Bool {
+			if value.Bool() {
+				tagName := field.Tag.Get("long")
+				warningStrs = append(warningStrs, fmt.Sprintf("--%s", strings.ToLower(tagName)))
+			}
+		value.SetBool(true)
```

Comment:
indentation issue (although it didn't cause `+lint` to fail which is odd)
```suggestion
			value.SetBool(true)
```